File: SoloLearn_CodeCoach/Easy/Popsicles/Popsicles.java
import java.util.Scanner;

public class Popsicles {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int siblings = input.nextInt();
        int popsicles = input.nextInt();

        System.out.println(popsicles % siblings == 0 ? "give away" : "eat them yourself");
    }
}

File: SoloLearn_CodeCoach/Easy/Popsicles/Popsicles.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    const string YES = "give away";
    const string NO = "eat them yourself";

    int siblings;
    int popsicles;

    cin >> siblings >> popsicles;

    cout << (popsicles % siblings == 0 ? YES : NO) << endl;
    return 0;
}
